### PR TITLE
Add gulpfile

### PR DESCRIPTION
**Pull request by lionelB**
and tweak npm script \o/

### CODE REVIEW

**Comment by lionelB** [comment]
cleaner version of #10

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+  mainJs:{
+    src:'src/kept.jsx',
+    dist: 'dist/js/kept.js'
+  }
+
+};
+
+
+
```
Comment (n1k0) [inline gulpfile.js]: nit: unecessary blank lines

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  gulp.watch([opt.outputFolder + '/**/*.*'])
+    .on('change', function(){
+      // trigger the live reload server
+      gulp.src('').pipe(connect.reload());
+    });
+});
+
+gulp.task('dist', ['assets', 'transpile', 'uglify']);
+gulp.task('default', ['server', 'watch']);
```
Comment (n1k0) [inline gulpfile.js]: nit: please add a CR at EOF

**Comment by n1k0** [comment]
Looks good. Thanks!

**Comment by n1k0** [comment]
Landed in 4877e9c96fc90ca0b1ea4fcb286182bc78cb83fe

**Pull request closed without merging**